### PR TITLE
Add toast message for send email button

### LINKED ISSUE #15: Add FE response handling for email notifications
### Description
- As an admin, I'd like to know if I was/wasn't able to email a user from the admin dashboard

### Acceptance Criteria
- Render a toast message saying "Email sent!" if request to `/send-email` goes through, or "Error: (what happened in layman's terms)" if request didn't go through

### PR DESCRIPTION
**Pull request by beebzz**
Fixes #15 and refactors profiles into separate component

### CODE REVIEW

File: components/admin-dashboard/profile.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      date: application.interviews.data[nextInterview].date
+    };
+    interviews.push(interview);
+  })
+  return interviews;
+};
+
+toast.configure();
+export default function Profile({ entry }) {
+  const sendEmailSuccessToast = () => {
```
Comment (zeeberry) [inline components/admin-dashboard/profile.jsx:23]: given this function doesn't really "send" but instead "shows" perhaps we name it `showEmailSuccessToast`?

File: components/admin-dashboard/profile.jsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    toast.success('Email sent! 🚀');
+  };
+
+  const sendEmailErrorToast = (status, error) => {
```
Comment (zeeberry) [inline components/admin-dashboard/profile.jsx:27]: same comment as above

File: components/admin-dashboard/profile.jsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+          )
+        })}
+      </div>
+      <button style={{ 'marginBottom': '1em' }} onClick={() => sendEmail(entry.account.email)}>Send Email</button>
+    </div>
+  )
+};
+
+Profile.propTypes = {
+  entry: PropTypes.shape({
```
Comment (zeeberry) [inline components/admin-dashboard/profile.jsx:68]: dear lord, this shape is pretty complicated. I think we need to either rethink the data schema or create a new endpoint at some point. nothing to do right now. I'm just thinking into the future.

**Review by zeeberry — APPROVED** [review]
added small non-blocking suggestion. LGTM!

File: components/admin-dashboard/profile.jsx (hunk 1 above)
Comment (beebzz) [reply components/admin-dashboard/profile.jsx:23]: Gotcha, I meant to convey that it was the success(/error) toast for the send email button but that's clearer

Comment (zeeberry) [reply components/admin-dashboard/profile.jsx]: oooh. i see. in that case another option could have been `showSendEmailSuccessToast`. The first word always being the action/verb its triggering.

Comment (zeeberry) [reply components/admin-dashboard/profile.jsx]: its fine the way it is tho. no need to change anything.
